src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer.Tests/Properties/launchSettings.json

```diff
@@ -0,0 +1,11 @@
+{
+  "profiles": {
+    "Personalizer.Tests": {
+      "commandName": "Project",
+      "environmentVariables": {
+        "Key": "Value",
+        "AZURE_TEST_MODE": "Record"
```

Comment:
@dwaijam can you clarify why are you adding this file?
Are you setting testframework in record mode everytime you run these tests?
We do not want tests to run in record mode during CI
If you want more information, please follow testFramework documentation as these settings are only meant during development of tests and recording of tests locally.

Reply:
removed